tests/Uri/UriRetrieverTest.php

```diff
@@ -339,6 +339,28 @@ public function testJsonSchemaOrgMediaTypeHack()
         $this->assertTrue($retriever->confirmMediaType($mock, 'http://json-schema.org/'));
     }
 
+    /**
+     * @expectedException \JsonSchema\Exception\InvalidSchemaMediaTypeException
+     */
+    public function testJsonSchemaOrgMediaTypeBlacklistUnknown()
+    {
+        $mock = $this->getMock('JsonSchema\Uri\UriRetriever', array('getContentType'));
+        $mock->method('getContentType')->willReturn('Application/X-Fake-Type');
+        $retriever = new UriRetriever();
+
+        $retriever->confirmMediaType($mock, 'http://iglucentral.com');
```

Comment:
Use example.com for example domains please.

Reply:
fixed.